### PR TITLE
gui: 'Out of Sync Items' modal tweaks (fixes #3306)

### LINKED ISSUE #3306: UI bug in "show items"
UI bug

screenshot_2016-06-15_03-41-30

### Version Information

Syncthing Version: v0.13.7
OS Version: Xubuntu 16.04
Browser Version: Firefox 47.0

### PR DESCRIPTION
**Pull request by norgeous**
see commit `...`

### CODE REVIEW

File: gui/default/assets/css/overrides.css [hunk 1]
```diff
@@ -93,7 +93,7 @@ ul+h5 {
 }
 
 .table td {
-    padding-left: 20px !important;
```
Comment (norgeous) [inline gui/default/assets/css/overrides.css]: wasn't sure why this was here, so I have commented it out. does anyone know?

**Comment by calmh** [comment]
What does it change?

**Comment by norgeous** [comment]
as in commit comment:
- Fix tooltip changing the cell / row width (don't put tooltip on td)
- Set column widths 2-6-4
- Fix Dark Theme progress bar (now readable)
- Fix Dark Theme modal alert-info header colour (now purple)
- Dark Theme header text now dark (for colourful ones)

**Comment by calmh** [comment]
Does this fix #3306?

**Comment by norgeous** [comment]
yes fixes #3306

**Comment by calmh** [comment]
Seems to work for me!

**Comment by calmh** [comment]
@st-review merge it please

gui: Improve layout of "out of sync" modal (fixes #3306)

**Comment by st-review** [comment]
:ok_hand: Merged as fb778218f5bc7c29b453459fc6226b68d23e6069. Thanks, @norgeous!

**Pull request closed without merging**